src/irsForms/ScheduleE.ts

```diff
@@ -117,10 +117,13 @@ export default class ScheduleE implements Form {
 
   l21 = (): MatrixRow => (
     zip3(this.l3(), this.l4(), this.l20())
-      .map(([x, y, z]) => (x ?? 0) + (y ?? 0) - (z ?? 0))
+      .map(([x, y, z]) => displayNumber((x ?? 0) + (y ?? 0) - (z ?? 0)))
   ) as MatrixRow
 
-  l22 = (): MatrixRow => this.f8582.deductibleRealEstateLossAfterLimitation()
+  // Deductible real estate loss from 8582, as positive number
+  l22 = (): MatrixRow =>
+    this.f8582.deductibleRealEstateLossAfterLimitation()
+      .map((x) => x === undefined ? undefined : Math.abs(x)) as MatrixRow
 
   l24 = (): number => sumFields(this.l21().filter((x) => x !== undefined && x > 0))
   l25 = (): number => (
```

Comment:
I think l25 should also be absolute value
![image](https://user-images.githubusercontent.com/8848557/117405912-34a3c000-aec1-11eb-952e-517f8abaffad.png)

Reply:
This point about royalties has me confused and I think the issue is deeper than just filling some blanks here. What do you think about splitting off royalties as a separate issue and limiting this implementation to just real estate income? I'm researching more about this royalty income and it seems like it's not even pertinent to real estate, but more about intellectual property or something else.

Reply:
yeah that makes sense, royalties are not really logically grouped with rental income

Reply:
Ok took out royalties, and added absolute value on that always negative field, and parenthetical if negative on the line below.

Reply:
Did you see the other comment I left above WRT vacation housing vs rentals?

Reply:
Yep, just implemented that test, see below.